packages/blitz/src/cli/commands/new.ts

```diff
@@ -0,0 +1,302 @@
+import {loadEnvConfig} from "../../env-utils"
+import prompts from "prompts"
+import path from "path"
+import chalk from "chalk"
+import hasbin from "hasbin"
+import {cliCommand} from "../index"
+import arg from "arg"
+import {AppGenerator, AppGeneratorOptions, getLatestVersion} from "@blitzjs/generator"
+import {runPrisma} from "../../prisma-utils"
+
+enum TForms {
+  "react-final-form" = "React Final Form",
+  "react-hook-form" = "React Hook Form",
+  "formik" = "Formik",
+}
+enum TLanguage {
+  "typescript" = "TypeScript",
+  "javascript" = "Javascript",
+}
+type TPkgManager = "npm" | "yarn" | "pnpm"
+enum TTemplate {
+  "full" = "full",
+  "minimal" = "minimal",
+}
```

Comment:
Let's use literal types or objects instead of enums — they compile to something complicated and there are some other problems with enums:
<img width="933" alt="Screenshot 2022-04-07 at 10 30 50" src="https://user-images.githubusercontent.com/9019397/162156145-02b85c26-1efa-4506-9da6-0428aacfb564.png">

We can do something like this instead:

```ts
const forms = {
  "react-final-form": "React Final Form",
  "react-hook-form": "React Hook Form",
  "formik": "Formik",
}
type Form = keyof typeof forms

const language = {
  "typescript": "TypeScript",
  "javascript": "Javascript",
}
type Lang = keyof typeof language

type TPkgManager = "npm" | "yarn" | "pnpm"
type Template = "full" | "minimal"
```